dds/DCPS/ReactorTask.h

```diff
@@ -47,6 +47,7 @@ struct OpenDDS_Dcps_Export ThreadStatusManager {
     // TODO(iguessthislldo): Add Participant GUID
   };
   typedef OPENDDS_MAP(String, Thread) Map;
+  typedef OPENDDS_MAP(String, double) LoadMap;
```

Comment:
Extend `Thread` with a `double load` to avoid parallel maps.  The code that publishes the thread status on the builtin topic should also publish the load.

Reply:
Will be done later.